In [1]:
import numpy as np

In [2]:
cp=1.16 # kJ/kg°C
m=0.96 # kg/s
ti=157 # °C
tf=98 # °C


In [8]:
q=m*cp*(tf-ti)
print(q,'KW')

-65.7024 KW


In [27]:
d=.12 # m
A=np.pi*(d/2)**2 # m2
f=.4526 # m3/kg
v=45 # m/s
m=A*v/f # kg/s
print(round(m,2),'kg/s')

1.12 kg/s


In [35]:
t_d=10 #dias
t_s=10*(24*3600) # s al cabo de 10 dias
m=12.27 # kg/s
x_s= 0.001
x_g=1-x_s
m1=f*x_g*(60*60*24)
m2=f*t_s*x_g
print(round(m1,2),'kg/dia')
print(round(m2,2),'kg/10 dias')

1059067.87 kg/dia
10590678.72 kg/10 dias


In [38]:
t_d=10 #dias
t_s=10*(24*3600) # s al cabo de 10 dias
m=12.27 # kg/s
x_s= 0.001
x_g=1-x_s
m1=f*x_s*(60*60*24)
m2=f*t_s*x_s
print(round(m1,2),'kg/dia')
print(round(m2,2),'kg/10 dias')

1060.13 kg/dia
10601.28 kg/10 dias


$
t(dias)=10 \ dias \\
t(s_{10 dias}) = 864,000 \ segundos \\
\dot{m} = 12.27 \frac{kg}{s} \\
x_s = 0.001 \\
\dot{m_g} = \dot{m} x_s*t(s_{10 dias}) = 10,601.28 \frac{kg}{10 dias}
$